camelot/view/qml_view.py

```diff
@@ -138,9 +139,17 @@ def unregister(self, gui_context_name):
 
     @QtCore.qt_slot(QtCore.QObject)
     def remove_model(self):
-        for context_id, model in list(self.models.items()):
+        for gui_context_name, model in list(self.models.items()):
             if is_deleted(model):
-                del self.models[context_id]
+                del self.models[gui_context_name]
+
+    @QtCore.qt_slot(list, int)
+    def model_selection_changed(self, selected_rows, current_row):
```

Comment:
tbd : dit lijkt me nt juist, de update vd selection moet in de afhandeling van de changeselection gebeuren

Reply:
![image](https://user-images.githubusercontent.com/17574296/173941265-4954c2df-df5c-48ca-b1ea-83ab350de50a.png)